reviews/controller.py

```diff
@@ -28,15 +29,37 @@ def retrieve_pull_requests(self, org: str, repository: str) -> Union[Table, None
 
     def update_pull_requests(self, org: str, repository: str) -> List[PullRequest]:
         """Updates repository models."""
-        pull_requests = self.client.get_pull_requests(org=org, repo=repository)
 
+        def _get_reviews(pull_request):
```

Comment:
the following line is missing type information in the function and the return type

```python
from typing import Dict
```

```suggestion
        def _get_reviews(pull_request: PullRequest) ->Dict[str, bool] :
```